pkg/assets/assets_generate.go

```diff
@@ -25,6 +25,7 @@ func main() {
 	dirs := map[string]string{
 		"/lokomotive-kubernetes": "../../assets/lokomotive-kubernetes",
 		"/components":            "../../assets/components",
+		"/examples":              "../../examples",
```

Comment:
I don't think `examples` should be part of the assets, as in the future, we want to use those assets to craft release archive, which will contain files required for installation. `examples` does not really belong there.

Reply:
I am not sure I understand: the default configs generated by `lokoctl config generate --platform aws` (or other syntax) need to come from the assets and need to be in the release. Or do you suggest the repo should have both examples and default configs in separate directories?